chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/ActivityController.java

```diff
@@ -438,10 +438,23 @@ public ActivityMetadataPage metadataForVersions(@RequestParam(required = false)
                 id, ActivityConcept.VERSION, startDate, endDate);
     }
 
+    @ApiOperation(value = "Get metadata about auditable records in the system for certification bodies.",
+            notes = "All parameters are optional and will default to the first page of ONC-ACB activity "
+                    + "with a page size of the maximum allowed. Page number is 0-based. Activities will be returned "
+                    + "with the most recent activity first.")
+    @RequestMapping(value = "/metadata/beta/acbs", method = RequestMethod.GET,
+    produces = "application/json; charset=utf-8")
+    public ActivityMetadataPage metadataForAcbs(@RequestParam(required = false) Long start,
+            @RequestParam(required = false) Long end, @RequestParam(required = false) Integer pageNum,
+            @RequestParam(required = false) Integer pageSize) throws JsonParseException, IOException, ValidationException {
+        return pagedMetadataManager.getCertificationBodyActivityMetadata(start, end, pageNum, pageSize);
+    }
+
     @ApiOperation(value = "Get metadata about auditable records in the system for certification bodies.",
```

Comment:
I think in the past we have added "DEPRECATED" to the method description, since Swagger does not display any deprecation information.